foodji_api/Api/Controllers/RecipesController.cs

```diff
@@ -74,4 +74,38 @@ public async Task<IActionResult> CreateRecipe([FromBody] RecipeDto recipe)
 
         return CreatedAtAction(nameof(GetRecipe), new {id = result}, new {});
     }
+    
+    [HttpPut]
+    [ProducesResponseType(StatusCodes.Status202Accepted)]
```

Comment:
J'ai mis 202 étant donné que ce n'était pas un POST mais bien un PUT

Reply:
Except you literally return a `CreatedAtAction`, my friend ;) - Aussi, Accepted, ça veut dire que l'action est pas finie. Un `Created` peut aussi se retourner pour certaines `PUT` action, selon la doc - je pense que dans le cas de notre replace au complet, c'est justifié. Sinon, 200Ok serait le standard.

> [200 OK](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200)
>
>    The request succeeded. The result meaning of "success" depends on the HTTP method:
>
>        GET: The resource has been fetched and transmitted in the message body.
>        HEAD: The representation headers are included in the response without any message body.
>        PUT or POST: The resource describing the result of the action is transmitted in the message body.
>        TRACE: The message body contains the request message as received by the server.
>
> [201 Created](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201)
>
>    The request succeeded, and a new resource was created as a result. This is typically the response sent after POST requests, or some PUT requests.
> [202 Accepted](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202)
>
>    The request has been received but not yet acted upon. It is noncommittal, since there is no way in HTTP to later send an asynchronous response indicating the outcome of the request. It is intended for cases where another process or server handles the request, or for batch processing.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status